<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GameBoard</class>
 <widget class="QMainWindow" name="GameBoard">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>972</width>
    <height>626</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>TIC TAK TOE</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow {
background-image: url(:/image/5599352.jpg);
background-position:center;
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="button_2_2">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>340</y>
      <width>211</width>
      <height>131</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>32</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="button_0_2">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>80</y>
      <width>211</width>
      <height>131</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>32</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="button_1_0">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>210</y>
      <width>211</width>
      <height>131</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>32</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="button_2_1">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>340</y>
      <width>211</width>
      <height>131</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>32</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="statusLabel">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>10</y>
      <width>551</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>statusLabel</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_0_0">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>80</y>
      <width>211</width>
      <height>131</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>32</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="resetButton">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>470</y>
      <width>291</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>RESET</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_1_1">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>210</y>
      <width>211</width>
      <height>131</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>32</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="button_0_1">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>80</y>
      <width>211</width>
      <height>131</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>32</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="button_1_2">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>210</y>
      <width>211</width>
      <height>131</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>32</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="button_2_0">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>340</y>
      <width>211</width>
      <height>131</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>32</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="ReturnButton">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>470</y>
      <width>291</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>RETURN MAIN WINDOW</string>
    </property>
   </widget>
   <widget class="QLabel" name="scoreLabel">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>30</y>
      <width>321</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>score</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
